frontend/src/general/simpleTable/SimpleTableRow.jsx

```diff
@@ -0,0 +1,31 @@
+import React from "react";
+import PropTypes from "prop-types";
+import { TableCell, TableRow } from "@material-ui/core";
+
+export default function SimpleTableRow(props) {
+  const { name, value, centerValue } = props;
+
+  let align = "left";
+  if (centerValue) {
+    align = "center";
+  }
```

Comment:
Instead of passing along a `centerValue` boolean why not just pass along the align property itself? Then have the flexibility of other alignments if we want, and you'll only have to test one case instead of 2.

If you decide to stick with this, though, you can replace it with a ternary operator `const align = centerValue ? "center" : "left";`

Reply:
af6a9eb o7